app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -12,6 +12,6 @@ class VerifyCsrfToken extends Middleware
      * @var array
      */
     protected $except = [
-        //
+        'expense/*'
```

Comment:
Is there a reason you didn't verify the /expense endpoints?

Reply:
I put that there to quickly test something and I remove it until the phase 2 PR